text-ui-test/input.txt

```diff
@@ -0,0 +1,18 @@
+todo borrow book
+
+deadline return book /by Sunday
+
+event project meeting /from Mon 2pm /to 4pm
+
+list
+
+todo
+
+deadline
+
+event
+
+deadline ah
+
+
+
```

Comment:
Wonderful use.

What would happen if a user enters an invalid command or tries to mark a task that does not exist?